js/scripts/three-class-config.js

```diff
@@ -845,7 +862,7 @@ module.exports = {
             encoding:         new Types.Enum('TextureEncodings', 'LinearEncoding'),
             version:          new Types.Int(0),
         },
-        construtorArgs: [ 'image', 'mapping', 'wrapS', 'wrapT', 'magFilter', 'minFilter', 'format', 'type', 'anisotropy' ],
+        //constructorArgs: [ 'image', 'mapping', 'wrapS', 'wrapT', 'magFilter', 'minFilter', 'format', 'type', 'anisotropy' ],
```

Comment:
Gotcha.  A couple loose thoughts:

image was not included because three.js expects an html image object, which is hard to represent in python land.  it could be interested to represent images in python as just an open file or file uri and then convert that to an image tag on the javascript end.  Definitely a project, and probably worth deferring for now

NOTE that ImageTexture is not an actual three.js type.  This is a custom type meant to make constructing image textures much easier and straight forward via the python interface.  Instead of polluting/altering the Texture class (and diverging from the three.js api's), I decided to build the custom subclass that allows making image textures.